guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc

```diff
@@ -0,0 +1,36 @@
+[id="variables-for-generation-of-conversion-data_{context}"]
+= Variables for Generation of Conversion Data
+
+Before you run the Ansible role to generate conversion data, configure values of the following required Ansible variables.
+
+{Project} imports most of the required Ansible variables from the `{ansible-namespace}.convert2rhel` role.
+However, some of the required variables are not imported.
+These variables are marked with an asterisk * in the table below.
+You must create those additional variables manually and assign them to the `{ansible-namespace}.convert2rhel` role.
+
+.Required variables for conversion
+[cols="20%,5%,20%",options="header"]
+|====
+| Name | Type | Intent and value
+| `{project-context}_server_url` * | string | URL of your {ProjectServer}
+| `{project-context}_username` * | string | Your admin login, such as `admin`
+| `{project-context}_password` * | string | Your admin password
+| `{project-context}_organization` * | string | Name of your organization
+| `{project-context}_rhel_wait_for_syncs` * | boolean | Must be set to `false`
```

Comment:
I wouldn't say `must be set`, it can be set to `true` but it kinda doesn't make sense for user to do it